### PR TITLE
travis: remove sudo: required

### PR DESCRIPTION
**Pull request by berendt**
The sudo keyword will be fully deprecated.

https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration

Signed-off-by: Christian Berendt <berendt@betacloud-solutions.de>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -47,4 +45,4 @@ after_failure:
 
 env:
   global:
-    secure: Q7ilx50Ch5DPNiSfTpEwrlrzGOXTFaKVoaGdKWHoxj5zf2+G3/pggCtW3ZTeuof0AtHjsnfG0f20Y+S+pwo9q+ksTa52UdUIBOXZZVeovGfQAaH23E+gxJwxHYdWwhSJAzpRzFKgr7XoZO+lwMFYun0sBCTk8lLG/nEMw37t3ks=
\ No newline at end of file
+    secure: Q7ilx50Ch5DPNiSfTpEwrlrzGOXTFaKVoaGdKWHoxj5zf2+G3/pggCtW3ZTeuof0AtHjsnfG0f20Y+S+pwo9q+ksTa52UdUIBOXZZVeovGfQAaH23E+gxJwxHYdWwhSJAzpRzFKgr7XoZO+lwMFYun0sBCTk8lLG/nEMw37t3ks=
```
Comment (leseb) [inline .travis.yml]: This is not related to your change

Comment (berendt) [reply .travis.yml]: That's right, that was my Vi. Does the added \n break anything?

Comment (leseb) [reply .travis.yml]: No but this is not part of your change so this should not be there :).
